{{cookiecutter.github_repository}}/ansible.cfg

```diff
@@ -0,0 +1 @@
+provisioner/ansible.cfg
```

Comment:
Vagrant runs in the ansible command while being in same folder :) So created a symlink